nhais-adaptor/adaptor/outgoing/interchange_adaptor.py

```diff
@@ -1,35 +1,47 @@
-from adaptor.outgoing.message_adaptor import MessageAdaptor
-from adaptor.outgoing.fhir_helpers.operation_definition import OperationDefinitionHelper as odh
+import adaptor.outgoing.message_adaptor as message_adaptor
+import adaptor.outgoing.fhir_helpers.fhir_finders as finders
 from edifact.models.interchange import Interchange
 from edifact.models.message import Messages
-from adaptor.outgoing.fhir_helpers.constants import ParameterName
+from adaptor.outgoing.fhir_helpers.fhir_creators import ParameterName
 
+"""
+An adaptor to take in fhir models and generate an edifact interchange
+"""
 
-class InterchangeAdaptor:
+
+def generate_recipient_from(nhais_cypher):
     """
-    An adaptor to take in fhir models and generate an edifact interchange
+    Generate the recipient cypher.
+    Will append '1' when the nhais cypher is 3 characters and '01' if 2 characters

```

Comment:
This is an edifact requirement and something that was explained to me by Mark;
`The “XX1” value if the Cipher of the NHAIS system. This can actually be 2 or 3 alpha characters. For example, “BD” is the Cipher for the Bedfordshire NHAIS System; “BRS” is the Cipher for the Avon NHAIS System. The “XX11” value is the EDI Trading Partner for the NHAIS system. This is the NHAIS system cipher with either “01” added on the end (for 2 character ciphers) or “1” added on the end (for 3 character ciphers). So, the Trading Partner for the Bedfordshire NHAIS system (BD) is “BD01” and the Trading Partner for the Avon NHAIS system (BRS) is “BRS1”.`

Basically we are able to deduce the recipient cypher from the nhais cypher. 
Do you think the function can be named a little better to explain this or add more detail in the docstring?

Reply:
I'd say a little more detail in the docstring to explain what we're doing/how we know what to do would make sense. This strikes me as one of those things somebody will come to and have no idea why it's being done, so it would be good to capture some of the above in the comments.

I think the function name makes sense as it is.

Reply:
👍 Done